tests/ecmp/test_inner_hashing.py

```diff
@@ -0,0 +1,190 @@
+# Summary: Inner packet hashing test
+# How to run this test: sudo ./run_tests.sh -n <tb name> -i <inventory files> -u -m group -e --skip_sanity -l info -c ecmp/test_inner_hashing.py
+
+import time
+import json
+import logging
+import tempfile
+import random
+
+from datetime import datetime
+
+import pytest
+import requests
+
+from tests.common.fixtures.ptfhost_utils import copy_ptftests_directory, change_mac_addresses   # lgtm[py/unused-import]
+from tests.ptf_runner import ptf_runner
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.disable_loganalyzer
```

Comment:
Why disabling loganalyzer? If the test causes other failures, disabling loganalyzer will miss these failures.

Reply:
Good point, removed the disable loganalyzer.